bin/embedding-sdk/release_utils.bash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+function update_readme() {
+  sed -i -E 's|(embedding-sdk-)[0-9.]+|\1'"$1"'|' enterprise/frontend/src/embedding-sdk/README.md
+  sed -i -E 's|(http://downloads.metabase.com/sdk/v)[0-9.]+|\1'"$1"'|' enterprise/frontend/src/embedding-sdk/README.md
+}
+
+function update_package_json_template() {
+  sed -i -E 's/("version": ").*"/\1'"$1"'"/' enterprise/frontend/src/embedding-sdk/package.template.json
+}
+
+$1 $2
```

Comment:
$1 calls the first argument which is the function name e.g.
```bash
release_utils.bash update_readme
```

$2 pass the argument to the function e.g.
```bash
release_utils.bash update_readme 0.1.6
```

`$1 $2` will expand to `update_readme 0.1.6`

And the passed argument could be accessed with $1 (first argument) in each function.